#! /bin/sh /usr/share/dpatch/dpatch-run
## DP: expose members

@DPATCH@
diff -r 5256b881c340 openscenegraph-2.0/OpenSceneGraph/include/osg/AnimationPath
--- a/OpenSceneGraph/include/osg/AnimationPath	Fri Sep 07 16:27:31 2007 +0200
+++ b/OpenSceneGraph/include/osg/AnimationPath	Sun Sep 09 02:09:32 2007 +0200
@@ -117,8 +117,6 @@ class OSG_EXPORT AnimationPath : public 
                 matrix.preMult(osg::Matrixd::translate(-_position));
             }
         
-        protected:
-
             osg::Vec3d _position;
             osg::Quat _rotation;
             osg::Vec3d _scale;
